StripeUICore/StripeUICore/Source/Elements/Factories/Address/AddressSectionElement.swift

```diff
@@ -334,8 +334,10 @@ import UIKit
                 return partialResult + [postalCode]
             }
         }
-        // Set the new address fields, including any additional fields
-        addressSection.elements = ([name] + [country] + [autoCompleteLine] + addressFields + [phone]).compactMap { $0 }
+        
+        let initialElements: [Element?] = [name, country, autoCompleteLine]
+        let phoneElement: [Element?] = [phone]
+        addressSection.elements = (initialElements + addressFields + phoneElement).compactMap { $0 }
```

Comment:
This `compactMap` call shouldn't be needed anymore (I don't know if it was needed in the first place), the arrays should be able to be concatenated directly.

Reply:
name, country etc can all be nil, they are properties on the class.

Reply:
Duh! For some reason I didn't realize they were optionals and thought `compactMap` was used to cast into the right type. My bad.